code_samples/.eslintrc

```diff
@@ -0,0 +1,13 @@
+{
+    "env": {
+        "mocha": true,
```

Comment:
Why do we need `mocha` in code samples?

Reply:
Good catch. Addressed in 13cc25c0903a7d4d671c49bd1751eac20f1b161e